load.php

```diff
@@ -168,6 +186,38 @@ function perflab_render_generator() {
 }
 add_action( 'wp_head', 'perflab_render_generator' );
 
+/**
+ * Don't load the performance modules if the core version of the module is available.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $module The name of the module.
+ * @return bool Whether to load module or not.
+ */
+function perflab_can_load_module( $module ) {
+	$module_load_file = plugin_dir_path( __FILE__ ) . 'modules/' . $module . '/can-load.php';
+
+	// If the load file does not exist, the feature is not in core and the module can be loaded.
```

Comment:
Similar to my other feedback, I'd rephrase the comments since the module being in core is not necessarily the only way that affects this. We should rather frame this more generically just around whether the module can be loaded by the current environment, and the module being in core is just one potential reason for that not being possible.
```suggestion
	// If the `can-load.php` file does not exist, assume the module can be loaded.
```

Reply:
Thanks for the feedback! Text changes updated.